tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -221,7 +221,12 @@ def tf_keras_random_data_run_with_callback(
             restore_best_weights=restore_weights,
         )
     else:
-        callback = tf.keras.callbacks.ProgbarLogger(count_mode="samples")
+
+        class CustomCallback(tf.keras.callbacks.Callback):
```

Comment:
As per https://github.com/tensorflow/tensorflow/issues/38618, manually constructing `ProgbarLogger` produces an infinity-related arithmetic error on recent versions of Keras. Accordingly, we swap in a simpler, valid callback here instead.